rc/tools/lint.kak

```diff
@@ -89,7 +89,16 @@ define-command \
                     '
                         BEGIN { OFS=":"; FS=":" }
 
-                        /:[1-9][0-9]*:[1-9][0-9]*:/ {
+                        # without column
+                        /^[^:]+:[1-9][0-9]*: / {
+                            $1 = ENVIRON["kak_bufname"]
+                            $2 += line_offset
+                            $2 = $2 FS "x";
```

Comment:
can we get away with an empty string instead of a magic 'x' here ?

Reply:
Yes, good idea: b83b576028685b6d160f1fb2e7ca2aae5c0f5a6f